lore/estimators/sklearn.py

```diff
@@ -63,6 +65,23 @@ class Regression(Base):
 
 
 class BinaryClassifier(Base):
+    def __init__(self, estimator):
```

Comment:
Why diverge the signature for `BinaryClassifier` different than `Base`, if it's a straight pass through?

Reply:
Is your thought that the signature for `BinaryClassifier` should be something like this?

`def __init__(self, estimator, eval_metric='log_loss', scoring_metric='auc')`

The way I had been thinking about it was that `BinaryClassifier` is a canned estimator with defaults already chosen. If a user passed in `scoring_metric='precision'`, that wouldn't change the actual scoring metric, just the label used for it. To change the actual metric, they would need to redefine `score`. In other words, I didn't want users to think that changing `scoring_metric` argument alone would actually change the `scoring_metric`. Agree though that it seems a bit clunky.